frontend/src/scenes/pipeline/hogfunctions/hogFunctionTestLogic.tsx

```diff
@@ -210,6 +210,42 @@ export const hogFunctionTestLogic = kea<hogFunctionTestLogicType>([
                 editor.revealLineInCenter(pos.lineNumber)
             }
         },
+
+        setTestResult: ({ result }) => {
+            if (result) {
+                setTimeout(() => {
+                    // First try to scroll the test results container into view
+                    const testResults = document.querySelector('[data-attr="test-results"]')
+                    if (testResults) {
+                        testResults.scrollIntoView({ behavior: 'smooth', block: 'start' })
+                    }
```

Comment:
This isn't a particularly react-y way to do this tbh. But its fine for now. Ideally you would use a ref on the div instead